config/external-routing.yml

```diff
@@ -24,32 +36,8 @@ metadata:
     kapp.k14s.io/change-group: istio.io/ingressgateway
 
 #! this gateway is used to expose CF services in the cluster, externally
```

Comment:
might be nice to include this in-line comment at the top of the gateway.lib.yml file too